scripts/mkcloud

```diff
@@ -1277,6 +1277,8 @@ Optional
         if there is a SLE12 node, deploy neutron-network role into the SLE12 node
     want_dvr=1 (default='')
         if DVR support should be turned on by neutron barclamp. Only works with openvswitch and vxlan.
+    want_docker=1 (default='')
+        Test docker image (uploading to glance, creating instance based on it etc.)
```

Comment:
i would just say deploy a docker compute node or sth. because testing it is optional and dependent on the `testsetup` step, isn't it?
maybe also mention that it only works for a sle12 node, or add a sanity check that won't let you `want_docker` without setting `want_sles12`